src/main/java/seedu/letsgethired/logic/commands/DeleteCommand.java

```diff
@@ -42,8 +42,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         InternApplication internApplicationToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deleteInternApplication(internApplicationToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_INTERN_APPLICATION_SUCCESS,
-                Messages.format(internApplicationToDelete)));
+        return new CommandResult(MESSAGE_DELETE_INTERN_APPLICATION_SUCCESS);
```

Comment:
Will this result in the user not knowing which Intern Application was just deleted?

Reply:
Yes it would, I could add it back if you guys want the behaviour to stay

Reply:
nvm added it back, feature could be useful